CharityPortal/charity_update/charity_test.go

```diff
@@ -0,0 +1,31 @@
+package charityupdate
+
+import (
+	"charity_portal/charity_update/database"
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWhenVisitingCharityUpdateThenShowsOrganization(t *testing.T) {
+	// Arrange
+	testCharity, _ := database.ReadCharityByEmailMock(context.Background(), "email")
+	router := CreateRouter(&dependencies{
```

Comment:
Shouldn't we use Compose function in tests too instead of direct dependencies injection?